hw2/04-background.js

```diff
@@ -0,0 +1,27 @@
+let timeInterval = setInterval(function () {
+  let randomColor1 = Math.floor(Math.random() * 255);
+  document.body.style.backgroundColor = "#" + randomColor1;
```

Comment:
Since randomColor1 is a variable this line should use template strings. When programmatically building up strings, use template strings instead of concatenation, as per the AirBnB style guide 6.3 [read about it here](https://github.com/airbnb/javascript#es6-template-literals)